certbot/certbot/_internal/lock.py

```diff
@@ -187,13 +187,13 @@ class _WindowsLockMechanism(_BaseLockMechanism):
     By default on Windows, acquiring a file handler gives exclusive access to the process
```

Comment:
I haven't reformatted this comment, but only removed the reference to `io.open`.